app/generic/verifier/facades/acapy_verifier_v1.py

```diff
@@ -44,44 +36,52 @@ async def create_proof_request(
 
     @classmethod
     async def get_credentials_for_request(cls, controller: AcaPyClient, proof_id: str):
+        pres_ex_id = pres_id_no_version(proof_id=proof_id)
```

Comment:
This is moved outside of the try, because it can raise a ValueError which should not be caught as general Exception